src/java/org/sakaiproject/feedback/tool/FeedbackTool.java

```diff
@@ -140,12 +143,23 @@ protected void doGet(HttpServletRequest request, HttpServletResponse response) t
         else if (!hasViewPermission){
             contactName = serviceName + " " + TEAM + " <" + sakaiProxy.getConfigString("mail.support", "") + ">";
         }
-        request.setAttribute("contactName", contactName);
+        setStringAttribute(request, "contactName", contactName);
 
         response.setContentType("text/html");
         request.getRequestDispatcher("/WEB-INF/bootstrap.jsp").include(request, response);
     }
 
+	private void setStringAttribute(HttpServletRequest request, String key, String value){
+		request.setAttribute(key, StringEscapeUtils.escapeJavaScript(value));
+	}
+
+	private void setMapAttribute(HttpServletRequest request, String key, Map map){
```

Comment:
It's probably worth keeping the generics on the Map, so you can never get a unexpected map that contains nested maps.